### ISSUE TITLE
Publish releases to Windows Package Manager (WinGet)

### ISSUE THREAD

**Issue by vedantmgoyal2009** [yarnpkg/yarn#8857]
With every new release of Yarn, use a GitHub workflow to update manifests at https://github.com/microsoft/winget-pkgs repository.

I can create a pull request if you would like. It would help in keeping manifests up-to-date at WinGet with every new release automatically.

**Comment by RDIL** [comment]
I don't think this is necessary, as the suggested way to use Yarn now is [Corepack](https://github.com/nodejs/corepack), which is bundled with Node.js.  All you have to do is run `corepack enable` in a command prompt with admin, and corepack will manage the version of Yarn being used per-project.

**Comment by vedantmgoyal2009** [comment]
So should we remove Yarn from WinGet?

**Comment by RDIL** [comment]
If it's already been added, I wouldn't remove it, but it's probably not worth it to add new versions.  It just complicates the already confusing setup of having a global Yarn which is installed through npm which then calls a newer Yarn version if it's present etc etc etc, it's much simpler to say enable Corepack which handles selecting the appropriate Yarn version where needed.

**Comment by vedantmgoyal2009** [comment]
My concern was that people might install older versions from WinGet if won't be updated there. That's why IMO it's best to remove the package from the repository or add a note in the documentation.

BTW, can you move this issue to berry repository so that it may grab more attention of maintainers as well as users 😉
